pkgs/applications/video/manim-community/default.nix

```diff
@@ -0,0 +1,141 @@
+{ lib
+, fetchPypi
+
+, buildPythonApplication
+, pythonOlder
+
+, cairo
+, ffmpeg_4
+, sox
+, xdg-utils
+, texlive
+
+, python
+, colour
+, numpy
+, pillow
+, scipy
+, tqdm
+, pydub
+, pygments
+, rich
+, pycairo
+, manimpango
+, networkx
+, setuptools
+, importlib-metadata
+, grpcio
+, grpcio-tools
+, watchdog
+, jupyterlab
+, moderngl
+, moderngl-window
+, mapbox-earcut
+}:
+
+let
+  # According to ManimCommunity documentation manim uses tex-packages packaged
+  # in a custom distribution called "manim-latex",
+  #
+  #   https://community.chocolatey.org/packages/manim-latex#files
+  #
+  # which includes another cutom distribution called tinytex, for which the
+  # package list can be found at
+  #
+  #   https://github.com/yihui/tinytex/blob/master/tools/pkgs-custom.txt
+  #
+  # these two combined add up to:
+  manim-tinytex = {
+    inherit (texlive)
+
+    # tinytex
+    scheme-infraonly amsfonts amsmath atbegshi atveryend auxhook babel bibtex
+    bigintcalc bitset booktabs cm dehyph dvipdfmx dvips ec epstopdf-pkg etex
+    etexcmds etoolbox euenc everyshi fancyvrb filehook firstaid float fontspec
+    framed geometry gettitlestring glyphlist graphics graphics-cfg graphics-def
+    grffile helvetic hycolor hyperref hyph-utf8 iftex inconsolata infwarerr
+    intcalc knuth-lib kvdefinekeys kvoptions kvsetkeys l3backend l3kernel
+    l3packages latex latex-amsmath-dev latex-bin latex-fonts latex-tools-dev
+    latexconfig latexmk letltxmacro lm lm-math ltxcmds lua-alt-getopt luahbtex
+    lualatex-math lualibs luaotfload luatex mdwtools metafont mfware natbib
+    pdfescape pdftex pdftexcmds plain psnfss refcount rerunfilecheck stringenc
+    tex tex-ini-files times tipa tools unicode-data unicode-math uniquecounter
+    url xcolor xetex xetexconfig xkeyval xunicode zapfding
+
+    # manim-latex
+    standalone everysel preview doublestroke ms setspace rsfs relsize ragged2e
+    fundus-calligra microtype wasysym physics dvisvgm jknapltx wasy cm-super
+    babel-english gnu-freefont mathastext cbfonts-fd;
+  };
+in
+buildPythonApplication rec {
+  pname = "manim-community";
```

Comment:
hmm, can it be just `manim`? It's so in [AUR](https://aur.archlinux.org/packages/manim/) or [MSYS2](https://packages.msys2.org/base/mingw-w64-manim). But if it can't be then fine.

Reply:
the problem is, we already have a package called `manim` in nixpkgs. unfortunately the existing package is a very outdated version and also not the community developed of manim. Maybe it could be worth considering removing the existing one in favor of manim-community.

Reply:
This is the last change request blocking the PR. How are these votes done? Or can a PR simply remove a package and the final reviewer gives the OK?

Reply:
cc @JohnAZoidberg